docs/sdks/sdk-ref-javascript.md

```diff
@@ -319,14 +320,38 @@ The `opts` object supports the following properties.
 | `useCookie` | `boolean` | Optional | Set this to `true` to tell the SDK to store the identity in cookie storage instead of local storage. You can still provide an identity using a first-party cookie if this value is false or not provided. | 
 | `callback` | `function(object): void` | Deprecated | The function that the SDK should invoke after validating the passed identity. Do not use this for new integrations. | N/A |
 
+#### Multiple Init Calls
+
+You can call the `init()` function any number of times.  In most cases, the  code will accept the latest value of a certain init parameter. For example, if init is called twice, and a different `baseUrl` is passed in each call, the `baseUrl` variable is updated to the value from the second call. 
```

Comment:
Might not hurt to somehow mention that you'll be listing out the params that get stored later (something like "see below").  I expected to see a list here when I was reading it.

Reply:
Good suggestion - I've linked the init parameters section here to make sure it's clear